api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/{submissionId}/getSignedUrl.ts

```diff
@@ -68,7 +68,6 @@ export function getSingleSubmissionURL(): RequestHandler {
 
     try {
       const getSurveySubmissionOccurrenceSQLStatement = getSurveySubmissionOccurrenceSQL(
-        Number(req.params.surveyId),
```

Comment:
do we need the surveyId for the query?  If not, then remove

Reply:
that is what I am doing here, previously it had surveyId for no reason - that is why I am removing it

Reply:
nevermind .. sorry